README.md

```diff
@@ -40,22 +40,22 @@ This plugin requires no configuration to use.
 ## Usage
 Write "value" into "key" in the "test" environment of "sitename".
 ```
-terminus secrets:set site.env key value
+terminus secrets:set sitename.env key value
```

Comment:
```suggestion
terminus secrets:set sitename.test key value
```
Updating it to test would match the verbiage above. Or the verbiage can be updated to ... in the "env" environment ...